### ISSUE TITLE
More control of Hyperparameters

### ISSUE THREAD

**Issue by antoninodimaggio** [antoninodimaggio/Hugging-Captions#3]
- Num of epochs, learning rate, temperature, top_p and so on
